### ISSUE TITLE
Requirements/Tests/UI folder contains tests that create issues in group1-pre

### ISSUE THREAD

**Issue by jaredmael** [CS673S15-Group1/Final_Project#19]
Since the creation of the tests folders, the new tests created work in ways that delete all info from the database after tests are completed.  test_project is no longer needed. test_projects now tests project related functions.  I asked for it to get it removed in my previous submission..   Could you please remove it and test_story should be removed as well because  It was made by my fellow test_creator but told me I could replace all of his previous tests. He has new ones and is waiting on mine to be complete so he can check for conflicts with mine (small ones that we know we have that he said he could fix quickly).
